src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: $hover-transition;
   }
+
+  &__link:hover {
+    color: #00acdc;
+  }
```

Comment:
Also in this situation
[checklist p3](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md)